packages/resin-universal-gosquared/src/node.js

```diff
@@ -0,0 +1,38 @@
+var Promise = require('bluebird')
+var GoSquared = require('gosquared')
+
+module.exports = function(gosquaredId, apiKey, debug) {
+	var visitor = null
+	function createVisitor(userId) {
+		if (visitor) return visitor
+		var gosquared = new GoSquared({
+			api_key: apiKey,
+			site_token: gosquaredId
+		})
+
+		if (userId) {
+			gosquared.createPerson(userId)
```

Comment:
What I'm saying is that you're not assigning the result of `createPerson`, the linked page doesn't make it clear to me this will have any effect.